src/app/DTT/select/index.js

```diff
@@ -136,7 +136,11 @@ const Write = ({
     newValue ? setIsFocused(true) : setIsFocused(false)
     onSendAnswer(prepareValueForSendingAnswer(newValue))
   }
-
+  const formatOptionLabel = ({ label, value }) => (
+    <div test-id={`${questionCode}-${value}`}>
```

Comment:
Put the test-id in Text component itself. No need to have an extra div wrapped around it. 
ie: 
      <Text test-id={`${questionCode}`>{label}</Text>

Reply:
Fixed in latest commit